app/src/main/res/layout/item_business.xml

```diff
@@ -1,44 +1,44 @@
-<android.support.v7.widget.CardView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     style="@style/CardView.Light"
-    android:layout_width="match_parent"
+    android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:layout_marginTop="8dp"
     android:layout_marginStart="8dp"
+    android:layout_marginTop="8dp"
     android:layout_marginEnd="8dp">
 
     <android.support.constraint.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="match_parent">
 
         <ImageView
             android:id="@+id/ivBusinessImage"
-            android:layout_width="0dp"
+            android:layout_width="298dp"
```

Comment:
Be careful of exact dimensions, Android has a lot of different sized phones and it might not look good on those phones or even when you rotate the phone and have it in landscape mode. This might be okay though depending on what you're doing, just be careful.